hack/dev-release.sh

```diff
@@ -3,14 +3,28 @@
 set -e
 set -u
 
-OPERATOR_SOURCE="./test/operator-hub/operator_source.yaml"
+INDEX_IMAGE="${INDEX_IMAGE:-quay.io/$OPERATOR_GROUP/$GO_PACKAGE_REPO_NAME-index:$BUNDLE_VERSION}"
 SUBSCRIPTION="./test/operator-hub/subscription.yaml"
 INSTALL_PLAN_PRIOR=service-binding-operator.v${BUNDLE_VERSION}
 
 sed -i -e "s,REPLACE_CSV_VERSION,service-binding-operator.v${BUNDLE_VERSION},g" ${SUBSCRIPTION}
 
-kubectl apply -f ${OPERATOR_SOURCE}
-./hack/check-operatorsource-pods.sh
+# Catalog source
+kubectl apply -f- << EOD
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: example-operators
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: $INDEX_IMAGE
+  displayName: Community Operators
+  publisher: Red Hat
+EOD
+
+./hack/check-catalogsource-pods.sh
```

Comment:
Since we are reworking things where, what is the purpose of this check? Do we really need it? IMHO, once we add the catalog source, we can immediately apply the subscription, wait for the operator to become available, and then eventually apply either ALM example or some other smoke test.

While we are here - how about an even more radical approach: `dev-release.sh` with an acceptance test that installs SBO and perform some binding? That would reduce the maintenance complexity a lot IMHO.